nestkernel/connection_manager.cpp

```diff
@@ -157,6 +158,13 @@ nest::ConnectionManager::set_status( const DictionaryDatum& d )
       "If structural plasticity is enabled, sort_connections_by_source can not "
       "be set to false." );
   }
+
+  updateValue< bool >( d, names::use_compressed_spikes, use_compressed_spikes_ );
+  if ( use_compressed_spikes_ and not sort_connections_by_source_ )
```

Comment:
Will this check work correctly also if `sort_connections_by_source_` is changed in the same call to `SetKernelStatus`?

Reply:
yes, it should. the previous 6 lines update the value of `sort_connections_by_source_` so here we can assume that has been set to the desired value, and if that value clashes with the one for `use_compressed_spikes_` we raise an error.